frontend/src/Providers.tsx

```diff
@@ -25,45 +24,19 @@ const getOidcConfig = async () => {
 };
 
 const onSigninCallback = (): void => {
-  window.history.replaceState({}, document.title, window.location.pathname);
+  // window.history.replaceState({}, document.title, window.location.pathname);
 };
 
 function AuthProviderWrapper({ children }: ProviderProps) {
-  const { isLoading, isError, data, error } = useQuery({
+  const { data } = useQuery({
     queryFn: getOidcConfig,
     queryKey: ['oidcConfig'],
   });
 
-  const SpinnerWrapper = (
```

Comment:
Is the spinning removed or did your change only make the wrapper obsolete?

Reply:
The change made the spinner and wrapper here obsolete.
On the login button is still a spinner.